api.go

```diff
@@ -533,10 +536,11 @@ type (
 	}
 
 	TranscodeFileReqOutputs struct {
-		Hls TranscodeFileReqOutputsHls `json:"hls,omitempty"`
+		Hls TranscodeFileReqOutput `json:"hls,omitempty"`
+		Mp4 TranscodeFileReqOutput `json:"mp4,omitempty"`
```

Comment:
FYI `omitempty` doesn't really work on structs. If it is important for them to be optional gotta make them pointers (but i guess not since those are requests not responses anyway)

Reply:
👍